claudia-lab/test/home-controller-test.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+describe('testing homeCtrl', function() {
+  beforeEach(() => {
+    angular.mock.module('claudiagram');
+    angular.mock.inject($controller => {
+      this.homeCtrl = new $controller('HomeController');
+    });
+  });
+
+  describe('testing initial properties', () => {
+    it('title should equal "Home"', () => {
+      expect(this.homeCtrl.title).toBe('Home');
+    });
+  });
+});
```

Comment:
@loomnugget tests are minimal but effective - you'll get better, with time, at adding in additional tests for edge cases